packages/app/src/components/CustomizePracticeCard.vue

```diff
@@ -7,8 +7,13 @@
         size="20px"
         round
         text-color="#C4C4C4"
-        icon="mdi-pencil"
-      />
+        icon="settings"
+        tooltip="some text"
+      >
+      <q-tooltip anchor="center right" self="center left" :offset="[10, 10]" content-class="bg-primary" content-style="font-size: 16px">
+        Customize Practice Session
```

Comment:
Maybe 'Customize Session' would be shorter and still descriptive enough. What do you think?